tests/integration_tests/instances.py

```diff
@@ -106,7 +106,9 @@ def push_file(
         # First push to a temporary directory because of permissions issues
         tmp_path = _get_tmp_path()
         self.instance.push_file(str(local_path), tmp_path)
-        assert self.execute("mv {} {}".format(tmp_path, str(remote_path))).ok
+        assert self.execute(
+            "mv {} {}".format(tmp_path, str(remote_path))
+        ).ok, f"Failed to push {tmp_path} to {remote_path}"
```

Comment:
All these lovely asserts can save a few characters by dropping the `.ok`. Think of the savings.